biojava-core/src/main/java/org/biojava/nbio/core/sequence/location/InsdcLocations.java

```diff
@@ -147,6 +147,10 @@ public GroupLocation(Location... subLocations) {
 			this(Arrays.asList(subLocations));
 		}
 
+		public GroupLocation(boolean isCircular, Location... subLocations) {
```

Comment:
The new parameter `isCircular` is not used.

Reply:
Good catch, thank you.
Done.